dlp/risk.py

```diff
@@ -0,0 +1,884 @@
+    subscription_path = subscriber.subscription_path(
+        project, subscription_id)
+    subscription = subscriber.subscribe(subscription_path)
+
+    # Set up a callback to acknowledge a message. This closes around an event
+    # so that it can signal that it is done and the main thread can continue.
+    job_done = threading.Event()
+
+    # Create helper function for unpacking values
+    def get_values(obj):
+        return int(obj.integer_value)
+
+    def callback(message):
+        try:
+            if (message.attributes['DlpJobName'] == operation.name):
+                # This is the message we're looking for, so acknowledge it.
+                message.ack()
+
+                # Now that the job is done, fetch the results and print them.
+                job = dlp.get_dlp_job(operation.name)
+                histogram_buckets = (
+                    job.risk_details
+                       .l_diversity_result
+                       .sensitive_value_frequency_histogram_buckets)
+                # Print bucket stats
+                for i in range(len(histogram_buckets)):
+                    print('Bucket {}:'.format(i))
+                    print('   Bucket size range: [{}, {}]'.format(
+                        (histogram_buckets[i]
+                            .sensitive_value_frequency_lower_bound),
+                        (histogram_buckets[i]
+                            .sensitive_value_frequency_upper_bound)))
+                    for value_bucket in histogram_buckets[i].bucket_values:
+                        print('   Quasi-ID values: {}'.format(
+                            map(get_values, value_bucket.quasi_ids_values)))
+                        print('   Class size: {}'.format(
+                            value_bucket.equivalence_class_size))
+                        for value in value_bucket.top_sensitive_values:
+                            print(('   Sensitive value {} occurs {} time(s)'
+                                   .format(value.value, value.count)))
+                # Signal to the main thread that we can exit.
+                job_done.set()
+            else:
+                # This is not the message we're looking for.
+                message.drop()
+        except Exception as e:
+            # Because this is executing in a thread, an exception won't be
+            # noted unless we print it manually.
+            print(e)
+            raise
+
+    # Register the callback and wait on the event.
+    subscription.open(callback)
+    finished = job_done.wait(timeout=timeout)
+    if not finished:
+        print('No event received before the timeout. Please verify that the '
+              'subscription provided is subscribed to the topic provided.')
+
+
+def k_map_estimate_analysis(project, table_project_id, dataset_id, table_id,
+                            topic_id, subscription_id, quasi_ids, info_types,
+                            region_code='USA', timeout=180):
```

Comment:
The default for region_code also needs to be added as "default=" to the corresponding command-line flag. Otherwise, the flag's default value of None will override this default when it's actually called.

Alternatively, if region_code can be safely omitted, maybe this should default to None in the first place?